src/DiaryHome.js

```diff
@@ -0,0 +1,19 @@
+import {useState} from "react";
+import {Col, Container, Row} from "react-bootstrap";
+import NewCardForm from "./components/NewCardForm";
+
+function DiaryHome() {
+    const [diaryNotes, setNotes] = useState(notes)
```

Comment:
Best practice is to use this as follows:

`const [diaryNotes, setDiaryNotes] = useState(notes)`

Setting method is declaring by adding `set`  to the front of the variable.